notebooks/MAST/Identifying-transient-candidates/identifying_highz_candidates.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>"Identifying Transient Candidates" should be part of the title since that's the filename. But I wonder if transient is the right word here? These objects are optically persistent, if difficult to detect </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>